src/main/java/kitty/Parser.java

```diff
@@ -0,0 +1,64 @@
+package kitty;
+
+public class Parser {
+    // Todo
+    public static boolean isTodoInputValid(String line) {
+        String taskName = line.substring(line.indexOf(" ") + 1);
+        if (taskName.equals("todo")) {
+            return false;
+        } else {
+            // Remove all white-spaces
+            taskName = taskName.replaceAll("\\s", "");
+            return !taskName.equals("");
+        }
+    }
+    public static String getTodoTaskName(String line) throws KittyException {
+        if (!isTodoInputValid(line)) {
+            throw new KittyException("Description of Todo cannot be empty!");
+        } else {
+            return line.substring(line.indexOf(" ") + 1);
+        }
+    }
+
+    // Deadline
+    public static boolean isDeadlineInputValid(String line) {
+        String taskName = line.substring(line.indexOf(" ") + 1, line.indexOf("/by"));
+        // Remove all white-spaces
+        taskName = taskName.replaceAll("\\s", "");
+        return !taskName.equals("");
```

Comment:
Perhaps there is another way to do this as it seems a bit complicated and might affect readability